visualization/dashboard/src/MLIDashboard/Interfaces/IExplanationDashboardProps.ts

```diff
@@ -37,6 +38,25 @@ export interface IExplanationDashboardProps {
     ) => Promise<any[]>;
 }
 
+export enum TelemetryLevels {
+    error = 'Error',
+    warning = 'Warning',
+    info = 'Info',
+    trace = 'Trace'
+}
+
+export type TelemetryLevel = 
```

Comment:
In the first iteration, it looks like Error is the only Telemetry type implemented right?

What do you have in mind for the other types?

Reply:
To be clear, the dashboard is not implementing anything for telemetry, thats what the hook is for, the hook passed in is the implementation. So Studio would pass in a function that did whatever made sense for Azure telemetry on an error (user errors in this case) versus a warning versus tracking user actions.